frontend/src/utils/YupSchema.ts

```diff
@@ -95,62 +95,21 @@ export const FinancialYear = Yup.object().shape({
   market: Financial.required(),
 });
 
-export const BuildingSchema = Yup.object().shape({
-  name: Yup.string()
-    .max(150, 'Name must be less then 150 characters')
-    .nullable(),
-  description: Yup.string()
-    .max(2000, 'Description must be less than 2000 characters')
-    .nullable(),
-  address: Address.required(),
-  latitude: Yup.number()
-    .min(-90, 'Invalid Latitude')
-    .max(90, 'Invalid Latitude')
+export const OccupancySchema = Yup.object().shape({
```

Comment:
The diff is really hard to read, but basically I realized that there was some duplication in the BuildingSchema, so it concats the relevant schemas instead of duplicating the logic from those schemas.